cmd/juju/machine/remove.go

```diff
@@ -133,12 +156,16 @@ func (c *removeCommand) Run(ctx *cmd.Context) error {
 	}
 	defer client.Close()
 
-	destroy := client.DestroyMachines
-	if c.Force {
-		destroy = client.ForceDestroyMachines
+	var results []params.DestroyMachineResult
+	if c.KeepInstance {
```

Comment:
Why encode all this here? You have already done this in the client and server code.

The whole point of encoding that in the api client is so you can just call DestroyMachinesWithParams on client for all.